### PR TITLE
Refs #67 Backport of inner-edition of a clearable FileField from creme 2.1

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

File: creme/creme_core/migrations/0052_v2_0__fakefilebag.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                ),
+            ],
+            options={
+                'ordering': ('name',),
+                'verbose_name': 'Test File bag',
+                'verbose_name_plural': 'Test File bags',
+            },
+            bases=('creme_core.cremeentity',),
+        ),
+    ]
```
Comment (genglert) [inline creme/creme_core/migrations/0052_v2_0__fakefilebag.py:33]: Move this in 0001_initial.py, in the test section.

File: creme/creme_core/tests/views/test_entity.py [hunk 2]
```diff
@@ -29,8 +30,11 @@
             CustomFieldString, CustomFieldDateTime,
             CustomFieldEnum, CustomFieldMultiEnum, CustomFieldEnumValue,
             FakeContact, FakeOrganisation, FakePosition, FakeSector,
-            FakeAddress, FakeImage, FakeImageCategory)
+            FakeAddress, FakeImage, FakeImageCategory,
+            FakeFolder, FakeDocument, FakeFileComponent, FakeFileBag
```
Comment (genglert) [inline creme/creme_core/tests/views/test_entity.py:34]: Missing ending comma

**Comment by genglert** [comment]
- Rename your branch "v2.0.17"
- Add a new section in the changelog: "== Version 2.0.17 Work in progress (not released -- this is a draft) =="
- Needs a better commit message (use the original message of the commit -- without <refs> -- we know that is a backport)

**Comment by joehybird** [comment]
Re-opened as release PR https://github.com/HybirdCorp/creme_crm-2.0/pull/3

**Pull request closed without merging**